Dockerfile

```diff
@@ -87,8 +87,12 @@ COPY ./api/os/os.proto /api/os/os.proto
 RUN protoc -I/api --go_out=plugins=grpc,paths=source_relative:/api os/os.proto
 COPY ./api/cluster/cluster.proto /api/cluster/cluster.proto
 RUN protoc -I/api --go_out=plugins=grpc,paths=source_relative:/api cluster/cluster.proto
-# Gofumports generated files to adjust import order
-RUN gofumports -w -local github.com/talos-systems/talos /api/
```

Comment:
is this intentional?

Reply:
well, no... Weird, I don't remember removing that line